package.json

```diff
@@ -10,6 +10,9 @@
   "engines": {
     "node": ">=8.0.0"
   },
+  "config": {
+    "wp_cmd": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js"
```

Comment:
Per my previous comment regarding errors on Windows 10. We *might* be able to replace this with:
`"node-options": "--max_old_space_size=4096"`
See: https://docs.npmjs.com/misc/config#node-options

If we can use that `node-options` configuration instead, then all other changes in this file might be reverted.  So, we can move back to just calling `webpack` in all commands.  Windows seems to mostly not like the `$npm_package_config_wp_cmd` environment variable.

Reply:
I just realized that I believe the `node-options` configuration (and NODE_OPTIONS env variable) are only supported in Node v8 and above.  So, if we go with this route, we should change our Requirements to require Node v8 or above (I'm fine with that, as I use Node 8 always anyways): https://github.com/DSpace/dspace-angular/#requirements

Reply:
In practice node 8 has been a requirement for nearly a year: https://github.com/DSpace/dspace-angular/commit/5554d4828b21f6e54a950ba9850e478ee94b15d8.

I'll update the docs in this PR.

Reply:
Hi @artlowel! I'm using your branch - atmire:preview-misc-fixes-2 (not the one in PR399) - and a new fresh setup (for the angular side). Just to note also that I'm using Linux CentOS 7 on my tests and I didn't had any issue building the project.